wrapper/src/descriptors.cc

```diff
@@ -21,4 +23,16 @@ namespace RDKit {
         auto computed_heap = new std::vector<double>(computed);
         return std::unique_ptr<std::vector<double>>(computed_heap);
     }
+
+    rust::String mol_formula(std::shared_ptr<ROMol> mol) {
+        return RDKit::Descriptors::calcMolFormula(*mol);
+    }
+
+    rust::u16 symmetrize_SSSR(std::shared_ptr<ROMol> mol){
```

Comment:
Rename to `symmetrize_sssr`, no capital characters in the function names

Reply:
sorry, I deleted `changelog` and modified the `Cargo.toml` just for testing. I want to publish to `crates.io` and test it in another project. I will close the pull request and reopen a new pull request.